lsif/server/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "lsif-server-http",
+  "description": "LSIF Server with an HTTP API",
+  "author": "Microsoft Corporation and Sourcegraph",
+  "license": "MIT",
+  "version": "0.1.0",
+  "engines": {
+    "node": "*"
```

```suggestion
    "node": ">=10.0.0"
```
if that is the requirement?

---

Node.js 8 works